frontend/src/lib/components/flows/utils.ts

```diff
@@ -54,13 +54,18 @@ export function getTypeAsString(arg: any): string {
 	if (arg === null) {
 		return 'null'
 	}
+	if (arg === undefined) {
+		return 'undefined'
+	}
 	return typeof arg
 }
 
 export function formatValue(arg: any) {
 	if (getTypeAsString(arg) === 'string') {
 		return `"${arg}"`
 	}
+
+	debugger
```

Comment:
never seen that before, how do use that

Reply:
I've removed the statement.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger